saleor/plugins/webhook/plugin.py

```diff
@@ -214,3 +231,123 @@ def page_deleted(self, page: "Page", previous_value: Any) -> Any:
             return previous_value
         page_data = generate_page_payload(page)
         trigger_webhooks_for_event.delay(WebhookEventType.PAGE_DELETED, page_data)
+
+    # Webhook payment functions
+
+    def __run_payment_webhook(
+        self,
+        event_type: str,
+        transaction_kind: str,
+        payment_information: "PaymentData",
+        previous_value,
+        **kwargs
+    ) -> "GatewayResponse":
+        if not self.active:
+            return previous_value
+
+        app = None
+        payment_app_data = from_payment_app_id(payment_information.gateway)
+
+        if payment_app_data is not None:
+            app = (
+                App.objects.for_event_type(event_type)
+                .filter(pk=payment_app_data.app_pk)
+                .first()
+            )
+
+        if not app:
```

Comment:
It is only a suggestion but I would add here some logger or/and insert more details for this error. Now it doesn't give us any feedback. We could add some details like: App ID, event type etc.